client/verta/verta/_uac/_workspace.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from verta._protos.public.uac import WorkspaceV2_pb2
+
+class Workspace(object):
+    """
+    Object representing a Workspace.
+
+    """
+
+    def __init__(self, conn, msg):
+        self.conn = conn
+        self.msg = msg
+        self.id = msg.id
+        self.org_id = msg.org_id
+        self.name = msg.name
+
+    @classmethod
+    def _create_proto(
+        cls, conn, name, org_id, permissions
+    ):
+        Message = WorkspaceV2_pb2.SetWorkspaceV2
+        msg = cls._create_msg(name, org_id, permissions)
+
+        response = conn.make_proto_request(
+            "POST",
+            "/api/v2/uac-proxy/organization/{}/workspaces".format(org_id),
```

Comment:
f-string if we could, please!

```suggestion
            f"/api/v2/uac-proxy/organization/{org_id}/workspaces",
```